components/emoji_picker/emoji_picker.stories.js

```diff
@@ -55,6 +55,7 @@ const recentlyUsedEmojis = [
 export const argsData = {
```

Comment:
Skin tone doesn't change when selecting in storybook, also we can probably change the dropdown to start on "Default" instead of "Light"

Reply:
![Screenshot 2023-05-30 at 10 20 59 AM](https://github.com/dialpad/dialtone-vue/assets/64808812/6063e885-009e-4da3-878a-89e1aef80add)

"Since they" = "Since the"